protocol/message.go

```diff
@@ -93,3 +127,55 @@ func (m *Message) Decrypt(cipherStr string, keys map[string]string) error {
 
 	return nil
 }
+
+func (m *Message) Encrypt(cipherStr string, keys map[string]string) error {
```

Comment:
Yup, just one string is probably better.  I do have an option to optionally pass in an `iv`, but in reality that's not a critical feature.